src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -55,6 +55,7 @@ public class InstancesController : ControllerBase
         /// </summary>
         /// <param name="instanceOwnerPartyId">the instance owner party id</param>
         /// <returns>list of instances</returns>        
+        [Authorize(Policy = "InstanceRead")]
```

Comment:
Is this correct.  Are we returning a instance here?

Reply:
removing the function